ssl/quic/quic_impl.c

```diff
@@ -4268,7 +4268,7 @@ SSL *ossl_quic_new_listener(SSL_CTX *ctx, uint64_t flags)
         goto err;
     }
 
-    /* TODO(QUIC SERVER): Implement SSL_LISTENER_FLAG_NO_ACCEPT */
+    /* TODO(QUIC FUTURE): Implement SSL_LISTENER_FLAG_NO_ACCEPT */
```

Comment:
If we're going to do this then we should probably retain somewhere a description of what that flag should do since you've removed that description from the public facing docs.